### PR TITLE
Small fixes in longevity-mv-si-2days.yaml

### PR DESCRIPTION
**Pull request by juliayakovlev**

### CODE REVIEW

File: tests/longevity-mv-si-4days.yaml [hunk 1]
```diff
@@ -1,18 +1,18 @@
-test_duration: 2880
-prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_profile_4mv_5queries.yaml ops'(insert=15,read1=1,read2=1,read3=1,read4=1,read5=1)' cl=QUORUM n=10000000 -port jmx=6868 -mode cql3 native -rate threads=10",
-                    "cassandra-stress user profile=/tmp/c-s_profile_3si_5queries.yaml ops'(insert=25,si_read1=1,si_read2=1,si_read3=1,si_read4=1,si_read5=1)' cl=QUORUM n=10000000 -port jmx=6868 -mode cql3 native -rate threads=10"
+test_duration: 5760
```
Comment (roydahan) [inline tests/longevity-mv-si-4days.yaml]: IN this case we need to change the file name as well from 2days to 4 days.

File: tests/longevity-mv-si-4days.yaml [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_profile_4mv_5queries.yaml ops'(insert=15,read1=1,read2=1,read3=1,read4=1,read5=1)' cl=QUORUM n=100000000 -port jmx=6868 -mode cql3 native -errors ignore -rate threads=10",
+                    "cassandra-stress user profile=/tmp/c-s_profile_3si_5queries.yaml ops'(insert=25,si_read1=1,si_read2=1,si_read3=1,si_read4=1,si_read5=1)' cl=QUORUM n=100000000 -port jmx=6868 -mode cql3 native -errors ignore -rate threads=10"
```
Comment (roydahan) [inline tests/longevity-mv-si-4days.yaml]: -errors ignore is a TEMP solution, we aren't sure we want this.

File: tests/longevity-mv-si-4days.yaml [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+                  "cassandra-stress user profile=/tmp/c-s_profile_3si_5queries.yaml ops'(insert=25,si_read1=1,si_read2=1,si_read3=1,si_read4=1,si_read5=1)' cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -errors ignore -rate threads=10",
+                  "cassandra-stress user profile=/tmp/c-s_profile_2si_2queries.yaml ops'(insert=10,si_p_read1=1,si_p_read2=1)' cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -errors ignore -rate threads=10"
                  ]
 n_db_nodes: 5
 n_loaders: 2
 n_monitor_nodes: 1
 nemesis_class_name: 'ChaosMonkey'
 nemesis_interval: 30
-nemesis_during_prepare: 'true'
+nemesis_during_prepare: 'false'
```
Comment (roydahan) [inline tests/longevity-mv-si-4days.yaml]: Another TEMP change, till we decide what to do.

File: tests/longevity-mv-si-4days.yaml (hunk 1 above)
Comment (juliayakovlev) [reply tests/longevity-mv-si-4days.yaml]: It's named "4days"

File: tests/longevity-mv-si-4days.yaml (hunk 3 above)
Comment (juliayakovlev) [reply tests/longevity-mv-si-4days.yaml]: @roydahan  now the time to decide. Do we want to run nemesis during preparing?

Comment (roydahan) [reply tests/longevity-mv-si-4days.yaml]: Yes

**Comment by juliayakovlev** [comment]
@roydahan  Changed according to comments. Please, review

**Pull request merged**